modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenParameter.java

```diff
@@ -27,7 +27,7 @@
     public boolean isFormParam, isQueryParam, isPathParam, isHeaderParam,
             isCookieParam, isBodyParam, hasMore, isContainer,
             secondaryParam, isCollectionFormatMulti, isPrimitiveType, isModel, isExplode;
-    public String baseName, paramName, dataType, datatypeWithEnum, dataFormat,
+    public String baseName, paramName, paramNameAlternative, dataType, datatypeWithEnum, dataFormat,
```

Comment:
I missed the reason to introduce `paramNameAlternative`. What about using vendor extensions instead similar to what we've done in other generators?

Reply:
Thanks for the hint. I changed it to `vendorExtensions`.